projects/engine-vscode/src/webview.ts

```diff
@@ -1,14 +1,18 @@
-import { PluginConnector, Profile, Message, PluginConnectorOptions, ExternalProfile } from "@remixproject/engine"
+import { PluginConnector, PluginConnectorOptions } from "@remixproject/engine"
+import { Message } from '../../utils/src/types/message'
+import { Profile, ExternalProfile } from '../../utils/src/types/profile'
+
 import { ExtensionContext, ViewColumn, Webview, WebviewPanel, window, Uri } from 'vscode'
 import { join, parse as parsePath } from 'path'
-import { promises as fs } from 'fs'
+import { promises as fs, watch } from 'fs'
 import { get } from 'https'
 import { parse as parseUrl } from 'url'
 
 interface WebviewOptions extends PluginConnectorOptions {
   /** Extension Path */
   context: ExtensionContext
   column?: ViewColumn
+  devMode?: boolean
```

Comment:
should `devMode`  be a property of `PluginConnectorOptions` ?

Reply:
Could be yes.